openapi.yaml

```diff
@@ -729,6 +764,25 @@ components:
             minItems: 1
             allOf:
               - $ref: "#/components/schemas/services"
+        references:
+          type: string
+          description: related external entities
+          nullable: true
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/reference"
+                required:
+                  - type
+                  - externalId
```

Comment:
I think we should make `externalId` a required field for all `oneOf` elements. @rafaelmaeuer @saerdnaer This way we can uniquely identify each element (`type` & `externalId`)?

Reply:
It depends on the use-case... Some systems like BR24 have not defined yet how their CRIDs/BRIDs will look like, so I wanted to give the users some flexibility.